workshop/web-vitals-lib/demo-npm-package/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "cwv-npm",
+    "version": "1.0.0",
+    "description": "Measure CWV with npm package web-vitals",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+      "start": "node index.js",
+      "test": "echo \"Error: no test specified\" && exit 1",
+      "build": "node_modules/.bin/browserify ./files/script.js -p esmify > ./files/bundle.js"
```

Comment:
In case if module isn't installed globally; but fixed to just "browserify" with latest commit.

Reply:
Because the command is executed by nodejs itself, it's doesn't matter if the module is installed globally or locally. 
With `node_modules/.bin/browserify` instead you are forcing nodejs to use only the locally installed version, which could lead to an error if a version is already installed globally and not additionally locally as well.